backend/main.go

```diff
@@ -199,6 +199,7 @@ func initialize(servicePort, databaseURL, redisURL, loggingURL, statsdURL, certP
 	r.Group(func(r chi.Router) {
 		r.Get("/logout", func(w http.ResponseWriter, r *http.Request) {
 			for _, cookie := range r.Cookies() {
+				svc.Log.Info("Removing cookie: "+cookie.Name+" "+cookie.Value, nil)
```

Comment:
Not at all--opens opportunity for session hijacking to bad actors but I'm curious why it only removes the refresh token and not the access token. I'll update it to not log value since that is not completely necessary for fixing the issue